Jenkinsfile

```diff
@@ -55,7 +55,7 @@ node('rhel8'){
   if(params.UPLOAD_LOCATION) {
     stage('Snapshot') {
       def filesToPush = findFiles(glob: '**.vsix')
-      sh "rsync -Pzrlt --rsh=ssh --protocol=28 *.vsix* ${UPLOAD_LOCATION}/snapshots/vscode-knative/"
+      sh "sftp -C ${UPLOAD_LOCATION}/snapshots/vscode-knative/ <<< \$'put -p *.vsix*'"
```

Comment:
I think you can rewrite it by using the filesToPush var otherwise you can delete ithat
`sh "sftp -C ${UPLOAD_LOCATION}/snapshots/vscode-knative/ <<< \$'put -p  ${filesToPush[0].path}'"`

Reply:
I have deleted filesToPush.